src/components/App.jsx

```diff
@@ -60,7 +65,7 @@ export const App = () => {
         )}
         <Main>
           <Content className="app-content">
-            {isQueryLoading(settingsQuery) ? (
+            {isQueryLoading(settingsQuery) || papersDefinitions.length === 0 ? (
```

Comment:
A noter pour plus tard, tu devrais gérer les erreurs que ce soit pour la settingsQuery mais aussi pour la papersDefinitions. Rester sur un loader c'est pas cool. Tu devrais créer une carte pour ça

Reply:
et le usePapersDefinitions devrait retourner la même API que le useQuery tant qu'à faire si c'est joauble

Reply:
> A noter pour plus tard, tu devrais gérer les erreurs que ce soit pour la settingsQuery mais aussi pour la papersDefinitions. Rester sur un loader c'est pas cool. Tu devrais créer une carte pour ça.
Et le usePapersDefinitions devrait retourner la même API que le useQuery tant qu'à faire si c'est jouable

C'est noté 👍